### PR TITLE
Activiti/Activiti#4086 persist project configurations

### PR DESCRIPTION
**Pull request by Giovanni007**
Ref: https://alfresco.atlassian.net/browse/AAE-10397

### CODE REVIEW

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-api/src/main/java/org/activiti/cloud/modeling/api/ProjectConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.cloud.modeling.api;
+
+public interface ProjectConfiguration {
+
+    String getId();
```
Comment (balsarori) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-api/src/main/java/org/activiti/cloud/modeling/api/ProjectConfiguration.java]: do we need to expose this? Seems to be specific to data persistence only. Is it possible to remove it here and only keep at the entity level?

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-api/src/main/java/org/activiti/cloud/modeling/api/ProjectConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    void setId(String id);
+
+    void setEnableCandidateStarters(Boolean enableCandidateStarters);
+
+    Boolean getEnableCandidateStarters();
+
+    void setProject(Project project);
```
Comment (balsarori) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-api/src/main/java/org/activiti/cloud/modeling/api/ProjectConfiguration.java:24]: Same here. Do we need to backward relation information?

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-api/src/main/java/org/activiti/cloud/modeling/api/ProjectConfiguration.java (hunk 1 above)
Comment (Giovanni007) [reply activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-api/src/main/java/org/activiti/cloud/modeling/api/ProjectConfiguration.java]: I think the id can be removed, I'll do it. Thanks

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-api/src/main/java/org/activiti/cloud/modeling/api/ProjectConfiguration.java (hunk 2 above)
Comment (Giovanni007) [reply activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-api/src/main/java/org/activiti/cloud/modeling/api/ProjectConfiguration.java:24]: In this case, I use the Project here https://github.com/Activiti/activiti-cloud/pull/839/files#diff-77fa1dfa81b04014c0bc5c380c3b59c95e40a0783e56e57d3eb06672b77b205eR133 so I can't remove it

**Review by balsarori — APPROVED** [review]
LGTM

**Pull request merged**